components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -733,4 +734,59 @@ TEST_F(BravePrefProviderTest, EnsureNoWildcardEntries) {
   provider.ShutdownOnUIThread();
 }
 
+TEST_F(BravePrefProviderTest, TestShieldsHttpsMigration) {
+  BravePrefProvider provider(
+      testing_profile()->GetPrefs(), false /* incognito */,
+      true /* store_last_modified */, false /* restore_session */);
+
+  auto* prefs = testing_profile()->GetPrefs();
+  std::string old_content_setting_path =
+      "profile.content_settings.exceptions.httpUpgradableResources";
+  std::string default_content_setting_path =
+      "profile.default_content_setting_values.httpUpgradableResources";
+  {
+    prefs->Set(old_content_setting_path, *base::JSONReader::Read(R"({
```

Comment:
same as above, we should set a wildcard content setting and then check to make sure it migrates to the default pref. Please follow the examples in the other migration tests

Reply:
done, thanks!